infrastructure/cli/pkg/providers/aws/aws.go

```diff
@@ -4,14 +4,31 @@ import (
 	"cli/pkg/kube"
 	"context"
 	"fmt"
+	"log"
```

Comment:
we shouldn't use log yet I think (we're trying hard not to use it anywhere short term so we can do one coherent pass at the output